tests/Doctrine/Tests/Models/Issue5989/Issue5989Person.php

```diff
@@ -10,6 +10,7 @@
  * @DiscriminatorMap({
  *      "person"    = "Issue5989Person",
  *      "manager"   = "Issue5989Manager",
+ *      "secretary"  = "Issue5989Secretary",
```

Comment:
Please use a different entity and a new inheritance tree to test your specific scenario